GPU/Directx9/ShaderManagerDX9.cpp

```diff
@@ -475,6 +479,26 @@ void ShaderManagerDX9::VSUpdateUniforms(int dirtyUniforms) {
 		VSSetFloatArray(CONST_VS_UVSCALEOFFSET, uvscaleoff, 4);
 	}
 
+	if (dirtyUniforms & DIRTY_DEPTHRANGE)	{
+		float viewZScale = gstate.getViewportZScale();
+		float viewZCenter = gstate.getViewportZCenter();
+
+		// Adjust for D3D projection matrix. We got squashed up to only 0-1, so we multiply
+		// the scale factor by 2, and add an offset.
+		// Given the way we do the rounding, the offset is probably mostly irrelevant as we cancel
+		// it afterwards anyway.
+		viewZScale *= 2.0f;
+		viewZCenter -= 32768.0f;
```

Comment:
I mistyped, I mean that the *integer* part of the offset is irrelevant.

Using 0.5 or 32767.5 here as the offset causes the depth in Phantasy Star to come out as 38023.496094 , that is, 0.5 off from the value we want. So for some reason that's not the right thing.